### ISSUE TITLE
Implement a timed event that broadcasts almost expiring CFPs etc. through Twitter

### ISSUE THREAD

**Issue by jfversluis** [jfversluis/CfpExchange#38]
Since the Twitter account is now in place, it might be nice to have it broadcast more than just "a new CFP was added". For instance, have it tweet each week what CFPs are about to expire or something. Or just random sometimes to advertise itself.

Maybe through an Azure Function?

**Comment by Jandev** [comment]
Great idea!

I guess we need to add a new column `ReminderSent (DateTime)`. This can be used to send out the tweets for expiring CFP's.
Something I dislike about bots is them bombing my timeline with dozens of tweets. I'd hate to see this happen to CFP Exchange. Therefore, the Function should run like every X hours. If there are CFP's expiring in the next 7 days and the `(DateTime.UtcNow - ReminderSent) < 7 days)`, it should send out a tweet, but only for 1 CFP!

If there are multiple CFP's expiring, in a couple of hours the next one will be picked up. And so on...

It's also possible to schedule a re-reminder. First reminder is 7 days before a CFP ends, the re-reminder is 3 days before the CFP ends.

**Comment by jfversluis** [comment]
I totally agree, I don't want to numb the users by sending dozens of tweets every day. I was indeed thinking once a week or something so that you are reminded of that one CFP you wanted to submit.

**Comment by staal-it** [comment]
What about Hangfire? --> https://www.hangfire.io/

**Comment by staal-it** [comment]
Yes, I believe you are correct. But that's not really a problem since there are no extra costs right? 

Shall I create a simple pull-request to get started with just one use-case or would you rather take a different approach?
